### PR TITLE
Note Outlook support

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
Also took the opportunity to clean up links and terms.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,7 +50,7 @@ Here's a 1-minute teaser video to give you a taste:
 
 The easiest way to try Script Lab is with the Tutorial file for Excel that walks you through the tool in a few steps. You can get it here: [script-lab-tutorial.xlsx](https://script-lab.azureedge.net/assets/documents/script-lab-tutorial.xlsx)
```
Comment (mscharlock) [inline README.md:51]:
```suggestion
The easiest way to try Script Lab for Excel, Word and Powerpoint is with the Tutorial file that walks you through the tool in Excel in just a few steps. You can get it here: [script-lab-tutorial.xlsx](https://script-lab.azureedge.net/assets/documents/script-lab-tutorial.xlsx)
```

File: README.md [hunk 2]
```diff
@@ -47,7 +50,7 @@ Here's a 1-minute teaser video to give you a taste:
...
 
-Alternatively, you can start from scratch and [install Script Lab from the Store](https://store.office.com/app.aspx?assetid=WA104380862).
+Alternatively, you can start from scratch and [install Script Lab from the Store](https://appsource.microsoft.com/product/office/WA104380862).
```
Comment (mscharlock) [inline README.md:53]:
```suggestion
Alternatively, you can start from scratch and [install Script Lab for Excel, Word, and Powerpoint from the Store](https://appsource.microsoft.com/product/office/WA104380862).
```

File: README.md [hunk 3]
```diff
@@ -2,38 +2,41 @@
...
 - **Run** the code in another pane beside the editor. Execution can include logic, API calls to Office, UI in the pane, and even output to a console. Every time you make a code change you can refresh the editor and run the new version in seconds.
 - **Share** your snippets through GitHub. If you create a snippet you'd like to share, you can use Script Lab to save it. Then send the link to someone else to try it and tweak it on their computer. The Import feature lets you load other people's snippets.
 
-You can [install Script Lab right now for free from the Office Store](https://store.office.com/app.aspx?assetid=WA104380862)! It works for Excel, Word, and PowerPoint. You'll need Office 2013 or later, Office Online, or Office for Mac.
+You can get [Script Lab](https://appsource.microsoft.com/product/office/WA104380862) right now for free from Microsoft AppSource! It works for Excel, Word, and PowerPoint. You'll need Office 2013 or later on Windows, Office on the web, or Office on Mac.
+
+If you're interested in developing for Outlook, you can get [Script Lab for Outlook](https://appsource.microsoft.com/product/office/WA200001603), also free from AppSource.
```
Comment (mscharlock) [inline README.md:39]: Worth mentioning what version of Office you need for it here? (Maybe not, I'm neutral on whether to include it)

**Comment by ElizabethSamuel-MSFT** [comment]
@TCourtneyOwen or another admin: This is ready to be merged. Thx!

**Pull request merged**